src/shims/foreign_items.rs

```diff
@@ -165,6 +166,7 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
             // Normally, this will be either `libpanic_unwind` or `libpanic_abort`, but it could
```

Comment:
There are some more functions a bit further up this code that should also have their ABI checked.